src/helpers/script_helpers.sh

```diff
@@ -14,6 +14,7 @@ function command_steps {
   echo_if_all_true "change_directory $(git_root)" "$RUN_IN_GIT_ROOT" "$IN_SUB_FOLDER"
   echo_if_all_true "stash_open_changes" "$STASH_OPEN_CHANGES" "$INITIAL_OPEN_CHANGES"
   steps
+  echo_if_all_true "set_previous_branch \$(expected_previous_branch)"
```

Comment:
That won't work either, those variables won't be correct at `$STEPS_FILE`'s compile time. I haven't really looked into the abort/continue architecture before, so I'll take a look and see how to get this to play nicely.